draft-yasskin-wpack-bundled-exchanges.md

```diff
@@ -127,15 +130,28 @@ metadata, and if one matches, load that request's response.
 
 ## Load a bundle's metadata {#semantics-load-metadata}
 
-This takes the bundle's stream and returns a map ({{INFRA}}) of metadata
-containing at least keys named:
+This takes the bundle's stream and returns either an error, an error with a
+fallback URL (where an error is a "format error" or a "version error"), or a map
```

Comment:
fallback URL == primaryUrl, is that right?  Felt it could be more clearly written out somewhere earlier than 3.3, step 21.